static/js/components/OpenAIPreferences.jsx

```diff
@@ -82,11 +83,12 @@ const OpenAIPreferences = () => {
           }
           label={profile.summary?.OpenAI?.active ? "Active" : "Inactive"}
         />
+        {profile.summary?.OpenAI?.active && <CustomizeOpenAIParameters />}
```

Comment:
maybe you want a `?' also between profile and summary in case the user doesn't have any summary settings yet? So that this works whether or not we add empty keys to the user preferences in the backend.